runners/basic_runner.py

```diff
@@ -53,11 +53,14 @@ def __str__(self):
 
     def runnable_status(self):
         """
-        Always execute the task if `waf --options=run_always` was used
+        Always execute the test task
+
+        If the run_modified option is present, then only the modified test
+        binaries will be run.
         """
         ret = super(BasicRunner, self).runnable_status()
         if ret == Task.SKIP_ME:
-            if self.generator.bld.has_tool_option('run_always'):
+            if not self.generator.bld.has_tool_option('run_modified'):
```

Comment:
I'm fine with always running the unit tests. I do that anyway :) @jpihl what do you think?